spec/model/recipe_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  describe 'User model' do
+    subject { Recipe.create(name: 'apple jam', preparation_time: 5, cooking_time: 4, description: 'hello', user_id: 1) }
+
```

Comment:
- [ ] All your `model` tests failed because you didn't create a `User` before asserting your tests. Remember that your test environment is different from your development environment and your production environment so please don't assume that the User will be available to you. Kindly implement this suggestion. 👍